pipeline/beam_tables.py

```diff
@@ -1004,10 +1004,11 @@ def _add_ip_metadata(self, date: str,
             'as_class': as_type,
             'country': country,
         }
-        if not metadata_values['country']:  # try Maxmind
-          (netblock, asn, as_name, as_full_name, as_type,
-           country) = maxmind_db.lookup(ip)
-          metadata_values['country'] = country
+        # Turning off maxmind data for now.
```

Comment:
I'd disable the maxmind_db construction above too to save resources and time.

Reply:
Ideally we should just inject a IP->country mapping object, rather than having to know about the implementations and fallback. No need to make that change though in this PR though.

Reply:
disabled